content/doc/tutorials/build-a-node-js-and-react-app-with-npm.adoc

```diff
@@ -111,6 +111,7 @@ GitHub account and then cloning this fork locally.
 * For macOS - `/home/Documents/GitHub/simple-node-js-react-npm-app`
 * For Linux - `/home/GitHub/simple-node-js-react-npm-app`
 * For Windows - `/home/Documents/GitHub/simple-node-js-react-npm-app`
+. In the *Script Path* field, specify the path to the Jenkinsfile (i.e. `jenkins/Jenkinsfile`)
```

Comment:
That change will prevent the user from taking the steps that the tutorial uses to show them a step-wise implementation of the Jenkinsfile by committing to their local copy of the repository.  It also makes the text in the next bullet incorrect.  The text in the next bullet says:

> You're now ready to begin creating your `Jenkinsfile`, which you'll be _checking into your locally cloned Git repository_.

If the user changes the Pipeline to use the `jenkins/Jenkinsfile`, they will immediately have the entire implementation of the Jenkinsfile without experiencing the ["initial"](https://www.jenkins.io/doc/tutorials/build-a-node-js-and-react-app-with-npm/#create-your-initial-pipeline-as-a-jenkinsfile), ["test"](https://www.jenkins.io/doc/tutorials/build-a-node-js-and-react-app-with-npm/#add-a-test-stage-to-your-pipeline), and ["deliver"](https://www.jenkins.io/doc/tutorials/build-a-node-js-and-react-app-with-npm/#add-a-final-deliver-stage-to-your-pipeline) commits that they are told to make during the tutorial.

```suggestion
```

Reply:
Oh yes! Sorry for the confusion. I am closing the pull request.

Reply:
Thanks for submitting a pull request!  So glad that you used the tutorial and took the time to suggest an improvement.